src/content/docs/infrastructure/prometheus-integrations/integrations-list/velero-integration.mdx

```diff
@@ -0,0 +1,88 @@
+---
+title: 'Velero integration'
+tags:
+  - Integrations
+  - Velero
+  - Backup
+  - Prometheus
+  - Kubernetes
+---
+
+import kubernetesVeleroDashboard from 'images/kubernetes_screenshot-crop_velero-dashboard.png'
+
+[Velero](https://velero.io/) is an open source tool for safely backing up and restoring resources in a Kubernetes cluster, performing disaster recovery, and migrating resources and persistent volumes to another Kubernetes cluster.
+
+
+Use New Relic to view a dashboard based on Prometheus metrics which helps you understand the Velero of your k8s cluster. With New Relic you can monitor:
```

Comment:
Maybe it should read "understand the success and performance of your Velero backup jobs for your k8s cluster"

Reply:
Thanks, updated